apps/accounts/helpers.py

```diff
@@ -0,0 +1,15 @@
+from django.conf import settings
+
+from jingo import register
+
+
+@register.function
+def fxa_config():
+    return {camel_case(key): value
```

Comment:
I becoming a fan of the second line of indents being 4 spaces over rather than where the `{` starts.